openedx/core/djangoapps/content/search/documents.py

```diff
@@ -567,6 +567,7 @@ def searchable_doc_for_container(
         Fields.org: str(container_key.library_key.org),
         # In the future, this may be either course_container or library_container
         Fields.type: DocType.library_container,
+        Fields.block_type: container_key.container_type,
```

Comment:
To check if it is "unit", "section", "subsection", etc..

Reply:
@rpenido  I added a comment to reflect this, but didn't change any code -- did you want me to?

Reply:
I added this comment just to give context to my change :smile: 
Everything is fine.